test/unit/hooks-worker-client-test.coffee

```diff
@@ -124,10 +134,10 @@ describe 'Hooks worker client', ->
         setTimeout ->
           hooksWorkerClient.stop (stopError) ->
             return done stopError if stopError
-            assert.isDefined runner.hookHandlerError
+            assert.ok runner.hookHandlerError
```

Comment:
I can't find `assert.ok` in Chai's docs; only `assert.isOk`. No big deal, but interesting.